generated/types.ts

```diff
@@ -24111,12 +24111,8 @@ export interface operations {
     };
     requestBody: {
       content: {
-        "application/json": {
-          /** An array of user `login`s that will be requested. */
-          reviewers?: string[];
-          /** An array of team `slug`s that will be requested. */
-          team_reviewers?: string[];
-        };
+        "application/json": Partial<{ [key: string]: any }> &
+          Partial<{ [key: string]: any }>;
```

Comment:
This looks like an incorrect handling of `anyOf` by https://github.com/octokit/openapi-types.ts. I'll make a follow up issue

Reply:
https://github.com/octokit/openapi-types.ts/issues/49